spec/support/api/schemas/tag.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
**Gem: `json-schema`**

This JSON Schema is used to validate the data structure of a tag returned in the GET tag API.